bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/message/Field.java

```diff
@@ -8,6 +8,8 @@
  */
 package org.openhab.binding.insteonplm.internal.message;
 
+import java.util.Objects;
```

Comment:
Please explain the changes made to this file.

Reply:
The class overrides `equals` but does not override `hashCode`. This breaks the contract whereby objects that are considered equal should have the same hashCode. There are other cases of this in the code base, but this one is the most problematic because `Field` is used as a key in a hashing structure in `XMLMessageReader`. LGTM provides more details on this error at https://lgtm.com/rules/2980070/